src/styles/utils/variables.scss

```diff
@@ -0,0 +1,11 @@
+$main-accent-color: #060b35;
+$secondary-color: #616070;
+$blue-accent-color: #00acdc;
+$card-border-color: #f3f3f3;
+$header-bg-color: #fff;
+$header-shadow-color: #0000000d;
+$title-hover-color: #34568b;
+$card-color: #fff;
+$header-logo-size: 40px;
```

Comment:
this size is used only for width and height of logo image, but it should be always a square, so I added a variable